Packs/Core/Playbooks/playbook-Suspicious_Hidden_User_Created.yml

```diff
@@ -0,0 +1,1263 @@
+              prefix:
+                value:
+                  simple: |
+                    Would you like to disable the following user?
+              suffix: {}
+      methods: []
+      format: ""
+      bcc:
+      cc:
+      timings:
+        retriescount: 2
+        retriesinterval: 360
+        completeafterreplies: 1
+        completeafterv2: true
+        completeaftersla: false
+      replyOptions:
+      - "Yes"
+      - "No"
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "31":
+    id: "31"
+    taskid: 3b615f1a-aba2-45db-8b12-26e38399944b
+    type: regular
+    task:
+      id: 3b615f1a-aba2-45db-8b12-26e38399944b
+      version: -1
+      name: Disable user account in AD
+      description: Disables the suspicious user in Active Directory.
+      script: '|||ad-disable-account'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "25"
+    scriptarguments:
+      username:
+        simple: ${Core.OriginalAlert.event.evtlog_target_username}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 270,
+          "y": 2390
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "32":
+    id: "32"
+    taskid: 040c833c-b457-462c-817b-66b06e05c1ea
+    type: regular
+    task:
+      id: 040c833c-b457-462c-817b-66b06e05c1ea
+      version: -1
+      name: Disable local user
+      description: Runs Powershell code on the affected host to disable the local user on the machine.
+      script: '|||core-run-script-execute-commands'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "25"
+    scriptarguments:
+      commands:
+        complex:
+          root: Core.OriginalAlert.event
+          accessor: evtlog_target_username
+          transformers:
+          - operator: concat
+            args:
+              prefix:
+                value:
+                  simple: powershell -Command Disable-LocalUser -Name "
+              suffix:
+                value:
+                  simple: '"'
+      endpoint_ids:
+        complex:
+          root: alert
+          accessor: agentid
+          transformers:
+          - operator: uniq
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 660,
+          "y": 2390
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "33":
+    id: "33"
+    taskid: 26ac1bd8-c555-4fe2-85f7-f93e90676d74
+    type: regular
+    task:
+      id: 26ac1bd8-c555-4fe2-85f7-f93e90676d74
+      version: -1
+      name: Terminate process group
```

Comment:
Please stick with our name convention for terminate causality (take it from @OmriItzhak playbooks)
It's important cause there is a bug that is fixed only in the next XDR release that will cause unexpected behaviour if it's the task name